### ISSUE TITLE
#Suggestion:  numbered cell outputs for easier notebook navigation (current 0.7.0 doesn't have it)

### ISSUE THREAD

**Issue by alexDeCastroAtGit** [andypetrella/spark-notebook#728]

**Comment by andypetrella** [comment]
We removed the cells execution number and instead the dirty cells are marked - if a cell A depends on cell B (variable in B used in A) and if cell B is reexecuted, hence the cell A will be flagged as dirty.
Which is better than tracking the numbers going fuzzy due to manipulations in the nb.

However, you're more talking about just numbering the outputs linearly I guess, which is fine and could easily done actually.

**Comment by alexDeCastroAtGit** [comment]
Yes, I mean actually numbering the outputs for IO stream navigation in the notebook. Thanks Andy
